### ISSUE TITLE
enum.values() minlen support

### ISSUE THREAD

**Issue by jsantino** [kelloggm/checker-framework#35]
enum.values() should return a array with a minlen of the number of enum values 

ie:

```
    public static enum Direction {
        NORTH, SOUTH, EAST, WEST
    };

    public static void enumValues() {
        Direction @MinLen(4)[] arr = Direction.values();
    }
```

_There is a test case in checker-framework/checker/tests/minlen/EnumValues.java_

**Comment by kelloggm** [comment]
Once the pull request I made to typetools/checker-framework that adds enum.values ArrayLen support to the value checker, this should just work.

**Comment by kelloggm** [comment]
It didn't just work. I added some code so that MinLen always checks method invocations for ArrayLen annotations, but that should probably just work more generally.

Anyway, this particular issue is fixed in cadd9f9f6b7f963b065ab0a2a6f1813bdd456d7f
